mandatory/2-logic-error.js

```diff
@@ -1,16 +1,16 @@
 // The syntax for these functions is valid but there are some errors, find them and fix them
 
 function trimWord(word) {
-  return wordtrim();
+  return word.trim();
 }
 
 function getStringLength(word) {
-  return "word".length();
+  return word.length;
 }
 
 function multiply(a, b, c) {
-  a * b * c;
-  return;
+  
+  return a*b*c;
```

Comment:
Good, but make sure you keep it tidy! I'd remove that extra line and ensure you have spaces between your variables and operands to improve readability :)